File: packages/expo-print/android/src/main/java/expo/modules/print/PrintPackage.java

package expo.modules.print;

import android.content.Context;

import java.util.Collections;
import java.util.List;

import org.unimodules.core.BasePackage;
import org.unimodules.core.ExportedModule;
import org.unimodules.core.ViewManager;

public class PrintPackage extends BasePackage {
  @Override
  public List<ExportedModule> createExportedModules(Context context) {
    return Collections.singletonList((ExportedModule) new PrintModule(context));
  }
}

File: packages/expo-print/android/src/main/java/expo/modules/print/PrintPackage.kt
package expo.modules.print

import android.content.Context
import expo.modules.core.BasePackage
import expo.modules.core.ExportedModule

class PrintPackage : BasePackage() {
  override fun createExportedModules(context: Context): List<ExportedModule> {
    return listOf(PrintModule(context))
  }
}
